src/openrct2/entity/EntityBase.h

```diff
@@ -23,23 +23,28 @@ enum class EntityType : uint8_t
     Null = 255
 };
 
-struct EntityBase
+struct EntitySpriteData
 {
-    EntityType Type;
-    EntityId Id;
-    int32_t x;
-    int32_t y;
-    int32_t z;
     // Width from centre of sprite to edge
-    uint8_t sprite_width;
+    uint8_t Width;
     // Height from centre of sprite to bottom
-    uint8_t sprite_height_negative;
+    uint8_t HeightMin;
```

Comment:
These aren't min and max they are negative and positive from the centre of the sprite to the extent of the image

Reply:
Sounds to me like ordinary bounds which are typically Center/-Min/+Max